src/crimson/os/seastore/randomblock_manager.cc

```diff
@@ -0,0 +1,667 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
80 character lines.

Reply:
The spacing and formatting in this file is highly inconsistent.  Please fix it and then resolve this comment once you have.

Reply:
Updated.